src/relayer/RelayerClientHelper.ts

```diff
@@ -119,18 +119,13 @@ export async function constructRelayerClients(
     );
   }
 
-  // We only use the API client to load /limits for chains so we should remove any chains that are not included in the
-  // destination chain list.
-  const destinationSpokePoolClients =
-    config.relayerDestinationChains.length === 0
-      ? spokePoolClients
-      : Object.fromEntries(
-          Object.keys(spokePoolClients)
-            .filter((chainId) => config.relayerDestinationChains.includes(Number(chainId)))
-            .map((chainId) => [chainId, spokePoolClients[chainId]])
-        );
-
-  const acrossApiClient = new AcrossApiClient(logger, hubPoolClient, destinationSpokePoolClients, config.relayerTokens);
+  // Determine which origin chains to query limits for.
+  const srcChainIds =
+    config.relayerOriginChains.length > 0
+      ? config.relayerOriginChains
+      : Object.values(spokePoolClients).map(({ chainId }) => chainId);
```

Comment:
Should we add a filter to ensure that no spokes are undefined? We do the check elsewhere

```suggestion
      : Object.values(spokePoolClients).filter(isDefined).map(({ chainId }) => chainId);
```

Reply:
Normally we check this when looping over chain IDs and then dereference the SpokePoolClient from an object. In this case the code iterates directly over the values so I don't think it can blow up in the same way, because we never create an entry for an undefined SpokePoolClient.